tasks/files-to-granules/README.md

```diff
@@ -0,0 +1,46 @@
+# @cumulus/files-to-granules
+
+[![Build Status](https://travis-ci.org/nasa/cumulus.svg?branch=master)](https://travis-ci.org/nasa/cumulus)
+
+This lambda function converts array-of-files input payloads into granule object output payloads.
+It is primarily intended to support the standard output of a [cumulus-process](https://github.com/nasa/cumulus-process-py) task,
+and convert that output into a granule object accepted as input by the majority of other Cumulus tasks.
+
+## Message configuration
+
+For more information on configuring a Cumulus Message Adapter task, see [the Cumulus workflow input/output documentation](https://nasa.github.io/cumulus/docs/workflows/input_output).
+
+### Config
+
+Config object fields:
+
+| field name | type | default | description
+| ---------- | ---- | ------- | -----------
+| input_granules | array\<object\> | (required) | Granules to which the files belong
```

Comment:
Is there a reason that this needs to be snake case instead of camel case?

Reply:
Backward compatibility with existing configs, mainly. Input for this task is mostly copy-paste from`move-granules`, which now uses similar schemas as PostToCmr.

Reply:
Same reason we store it in `$.meta.input_granules`? The cumulus message has a mix of cases. Do you think I should change to camel case and we should start moving towards that for the Cumulus message as a whole? Just wondering about that would mean breaking people's stuff, but I guess it would mainly be workflow config updates for any message adapter tasks.

Reply:
Any of our tasks would be easily reconfigured with config updates, and any non-CMA tasks wouldn't need to change their output with config updates. Long as we don't change where our outputs go I think it's config-only changes.

Reply:
Since this is a new task, it shouldn't break existing workflows.  I'll defer to you, but I don't see a reason to carry over that legacy issue into a new task.

Reply:
Changed 👍